ConceptMap/ConceptMap-Genomics-primary-sample-state.json

```diff
@@ -0,0 +1,141 @@
+  "targetUri": "http://snomed.info/sct?fhir_vs=ecl/%3C%20260787004%20",
+  "group": [
+    {
+      "target": "http://snomed.info/sct",
+      "element": [
+        {
+          "code": "DNA-stabiliser",
+          "display": "DNA Stabiliser",
+          "target": [
+            {
+              "code": "706052002",
+              "display": "Evacuated blood collection tube (physical object)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "edta",
+          "display": "EDTA",
+          "target": [
+            {
+              "code": "702282003",
+              "display": "Non-evacuated blood collection tube, EDTA",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "ffpe-curls/scrolls",
+          "display": "FFPE Curls/Scrolls",
+          "target": [
+            {
+              "equivalence": "unmatched"
+            }
+          ]
+        },
+        {
+          "code": "ffpe-slides",
+          "display": "FFPE Slides",
+          "target": [
+            {
+              "equivalence": "unmatched"
+            }
+          ]
+        },
+        {
+          "code": "fresh-cells",
+          "display": "Fresh Cells",
+          "target": [
+            {
+              "code": "706049005",
+              "display": "Blood tube (physical object)",
+              "equivalence": "subsumes"
+            }
+          ]
+        },
+        {
+          "code": "fresh-tissue",
+          "display": "Fresh Tissue",
+          "target": [
+            {
+              "code": "434464009",
+              "display": "Tissue cassette (physical object)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "lithium-heparin",
+          "display": "Lithium Heparin",
+          "target": [
+            {
+              "code": "702284002",
+              "display": "Non-evacuated blood collection tube, lithium heparin (physical object)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "other",
+          "display": "Other",
+          "target": [
+            {
+              "code": "260787004",
+              "display": "Physical object (physical object)",
```

Comment:
Might need higher level concept such as specimen state or specimen condition (neither in SNOMED CT)